ionic/src/app/switch/switch.page.scss

```diff
@@ -43,4 +43,7 @@
 }
 .blue {
     color: #11c1f3;
+}
+.yellow{
```

Comment:
create scss variables for above css properties.
like color, padding, margin etc .